tests/end-to-end/ui/06-messaging.js

```diff
@@ -199,7 +199,7 @@ function messageActionsTest() {
 				it('delete the message', () => {
 					mainContent.selectAction('delete');
 					mainContent.popupFileConfirmBtn.click();
-					browser.waitForVisible('.sweet-overlay', 3000, true);
+					flexTab.sweetAlertOverlay.waitForVisible(3000, true);
```

Comment:
`sweetAlertOverlay` could be part of `page` object don't you think? Cuz it's more global then related with flexTab component.